diesel/src/expression/sql_literal.rs

```diff
@@ -20,12 +21,50 @@ pub struct SqlLiteral<ST> {
 }
 
 impl<ST> SqlLiteral<ST> {
+    #[doc(hidden)]
     pub fn new(sql: String) -> Self {
         SqlLiteral {
             sql: sql,
             _marker: PhantomData,
         }
     }
+
+    /// Bind a value for use with this SQL query. This function should be used
+    /// with care, as Diesel cannot validate that the value is of the right type.
```

Comment:
Ironically I originally had that, and then started to go down a path where `.bind` appended the actual bind parameter SQL, and then decided that it was a bad API and scrapped it and forgot to add the docs back